dashboards/risk_investigation.xml

```diff
@@ -228,14 +230,14 @@ I've left the input toggle commented out so you can add it in if you'd like. -->
           </progress>
         </search>
         <drilldown>
-          <condition match="match('click.name2', &quot;comment&quot;)">
-            <link target="_blank">$row.ir_link|n$</link>
-          </condition>
+        <condition match="match('click.name2', &quot;ir_link&quot;)">
+            <link target="_blank">/app/SplunkEnterpriseSecuritySuite/incident_review?earliest=-30d&amp;latest=now&amp;search=event_hash%3D$row.event_hash$</link>
```

Comment:
Nice! Hope you don't mind me chipping in.

Small efficiency uptick if we retain the logic that's dynamically limiting the URL timerange to the notable
e.g.
```
<link target="_blank">/app/SplunkEnterpriseSecuritySuite/incident_review?earliest=$row.earliest$&amp;latest=$row.latest$&amp;search=event_hash%3D$row.event_hash$</link>
```
Though I'm wondering if we want to be using the `_time` field instead of `orig_time` for the earliest/latest eval (approx line 198)?

Reply:
I suppose we could switch back,  I had issues with the earliest and latest not populating and the url would just not be generated in that case. Since the drill down specifies that it's for 30 days I set it to -30 static to match and didnt have an issue. I should be able to figure out the earliest/latest time issue if that's what you all would prefer.

Reply:
I think that may be due to the current eval for latest/earliest fields (line ~198) is using `orig_time` - it assumes that the notable itself provides a _time value.

If that was changed to this
 `| eval latest=_time + 1800, earliest=_time - 1800` 

or this (if we wanted to retain orig_time for any future logic):
 `| eval latest=coalesce(orig_time, _time) + 1800, earliest=coalesce(orig_time, _time) - 1800`

do you experience the same issue?

The dynamic URL (assuming it works 😄) will load slightly faster, particularly in large environments with lots of notables, but static -30d definitely works too 👍

Reply:
Just made and tested those changes in our deployment and it works with the eval and _time change, thanks for the tip! 👍